tests/discovery/test_discovery.py

```
@@ -78,41 +53,20 @@ def wrap_signpost_uri(uri: str) -> str:
-    thread = threading.Thread(target=httpd.serve_forever)
-    thread.start()
-
-    yield
-
-    httpd.shutdown()
-    thread.join()
+@pytest.mark.fixture("httpd_server_base")
+def test_discovery_cases(httpd_server_base: str) -> None:
+    assert httpd_server_base
+    assert re.match(
+        r"^https?://[\w.-]+(?::\d+)?/?$", httpd_server_base
```

Ruff

Trailing comma missing

Add trailing comma

(COM812)